frontend/src/metabase-types/api/query.ts

```diff
@@ -123,14 +123,6 @@ type Value = null | boolean | StringLiteral | NumericLiteral | DatetimeLiteral;
 type OrderableValue = NumericLiteral | DatetimeLiteral;
 
 type RelativeDatetimePeriod = "current" | "last" | "next" | number;
-export type RelativeDatetimeUnit =
```

Comment:
This type became redundant since we added `DateTimeAbsoluteUnit`, which has the same values although the name makes more sense